README.md

```diff
@@ -16,12 +16,39 @@ The functionality is to map all the requests paths to generic endpoint in Turvis
    docker-compose up -d
    ```
 
-## Changes
+# Changes
+## Feature/6, Feature/9
 - Project setup with: 
   - node.js using typescript
   - prettier formatting
   - Lint static checks
   - fastify node.js server
   - Dockerfile for building the image
   - docker-compose for running the service
-- two endpoints, `GET turvis:8060/ruuter-incoming`and `POST turvis:8060/ruuter-incoming`, for accepting calls from ruuter.
\ No newline at end of file
+- two endpoints, `GET turvis:8060/ruuter-incoming`and `POST turvis:8060/ruuter-incoming`, for accepting calls from ruuter.
+
+## feature/7
+- turvis and ruuter setup with one composer file. \
+   ```
+   cd integration
+   make start
+   ```
+   By default it will build ruuter image from github main branch, copies the modified `application.yml` file to container and maps to `integration/configs/DSL` directory so that new queries can be modified outside of docker container itself.
```

Comment:
```suggestion
   By default it will build ruuter image from github main branch, copies the modified `application.yml` file to container and maps to `integration/configs/DSL` directory so that new queries can be modified outside of docker container itself.
```
```suggestion
   By default it will build Ruuter image from github main branch, copies the modified `application.yml` file to container and maps to `integration/configs/DSL` directory so that new queries can be modified outside of docker container itself.
```